src/Cli/dotnet/BuiltInCommandsCatalog.cs

```diff
@@ -158,7 +159,11 @@ public static class BuiltInCommandsCatalog
             ["internal-reportinstallsuccess"] = new BuiltInCommandMetadata
             {
                 Command = InternalReportinstallsuccess.Run
-            }
+            },
+            ["workload"] = new BuiltInCommandMetadata
+            {
+                Command = WorkloadCommand.Run
```

Comment:
Are we tracking setting the DocLink field here when we have docs?

Reply:
tracked in #16267